src/pages/workspace/WorkspacesListRow.tsx

```diff
@@ -0,0 +1,168 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import {SvgProps} from 'react-native-svg';
+import Avatar from '@components/Avatar';
+import Icon from '@components/Icon';
+import * as Illustrations from '@components/Icon/Illustrations';
+import Text from '@components/Text';
+import ThreeDotsMenu from '@components/ThreeDotsMenu';
+import withCurrentUserPersonalDetails, {WithCurrentUserPersonalDetailsProps} from '@components/withCurrentUserPersonalDetails';
+import useLocalize from '@hooks/useLocalize';
+import * as PersonalDetailsUtils from '@libs/PersonalDetailsUtils';
+import {AvatarSource} from '@libs/UserUtils';
+import useThemeStyles from '@styles/useThemeStyles';
+import CONST from '@src/CONST';
+import {Policy} from '@src/types/onyx';
+
+type WorkspacesListRowProps = WithCurrentUserPersonalDetailsProps & {
+    /** Name of the workspace */
+    title: Policy['name'];
```

Comment:
To keep code in the project consistent I would use `Pick<Policy, 'name'> instead - this is what we have been using util now.